### PR TITLE
DOCS-1667-3.18-early-preview

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
CE 3.18

Issue:
https://tigera.atlassian.net/browse/DOCS-1667

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.18/releases.json [hunk 1]
```diff
@@ -1,259 +1,263 @@
 [
   {
-    "title": "master",
+    "title": "v.3.18.0",
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.18/releases.json]: Bad find & replace: `"v.3.18.0"` should be `v3.18.0`

File: calico-enterprise_versioned_docs/version-3.18/releases.json [hunk 2]
```diff
@@ -1,259 +1,263 @@
...
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.30.5",
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.18/releases.json]: should be v1.31.0

File: calico-enterprise_versioned_docs/version-3.18/releases.json [hunk 3]
```diff
@@ -1,259 +1,263 @@
...
       "registry": "quay.io"
     },
     "calico": {
-      "minor_version": "v3.26",
+      "minor_version": "v3.25",
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.18/releases.json]: Please confirm with @mgleung  what this version should be. 
This should be coming and/or be in sync with calico-private/node/Makefile: v3.26
https://github.com/tigera/calico-private/blob/release-calient-v3.18/node/Makefile#L45

File: calico-enterprise_versioned_docs/version-3.18/releases.json [hunk 4]
```diff
@@ -1,259 +1,263 @@
...
+        "version": "v.3.18.0"
       },
       "prometheus": {
         "image": "tigera/prometheus",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "coreos-prometheus": {
-        "version": "v2.42.0"
+        "version": "v2.43.1"
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.18/releases.json]: This should come from https://github.com/tigera/prometheus-docker/blob/release-calient-v3.18/Makefile#L16C29-L16C29
or https://github.com/tigera/calico-private/blob/release-calient-v3.18/calico/_data/versions.yml
which is v2.45.0

File: calico-enterprise_versioned_docs/version-3.18/releases.json [hunk 5]
```diff
@@ -1,259 +1,263 @@
...
-        "registry": "quay.io"
-      },
-      "csi-node-driver-registrar": {
-        "image": "calico/node-driver-registrar",
-        "version": "master",
+        "image": "tigera/csi",
+        "version": "v.3.18.0",
         "registry": "quay.io"
       }
+    },
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.18/releases.json]: Indentation seems wrong.

File: calico-enterprise_versioned_docs/version-3.18/_includes/release-notes/_v3.18.0-release-notes.mdx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+31 August 2023
+
+### What's new {#whats-new-3.18.0}
+
+
+### Known issues {#known-issues-3.18.0}
+
+* Upgrading to {{prodname}} 3.18.0 on Rancher/RKE from {{prodname}} 3.13.0 currently requires manually terminating the calico-node container for an upgrade to proceed.
+* Multi-cluster management of managed clusters with version < v3.18.0 is currently not working.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.18/_includes/release-notes/_v3.18.0-release-notes.mdx]: This one should be deleted right? @caseydavenport

Comment (caseydavenport) [reply calico-enterprise_versioned_docs/version-3.18/_includes/release-notes/_v3.18.0-release-notes.mdx]: Yep, once those cherry-picks I just sent you are merged.

File: calico-enterprise_versioned_docs/version-3.18/releases.json (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18/releases.json]: oof! Fixed.

File: calico-enterprise_versioned_docs/version-3.18/releases.json (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18/releases.json (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18/releases.json]: Changed to 2.45.0

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18/releases.json:190]: Changed to v2.45.0

File: calico-enterprise_versioned_docs/version-3.18/_includes/release-notes/_v3.18.0-release-notes.mdx (hunk 6 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18/_includes/release-notes/_v3.18.0-release-notes.mdx]: Deleted mcm bullet.

File: calico-enterprise_versioned_docs/version-3.18/releases.json (hunk 3 above)
Comment (mgleung) [reply calico-enterprise_versioned_docs/version-3.18/releases.json]: this is still based on 3.25.1. We have a large PR that I haven't been able to bank yet that will catch everything up to 3.26.1 but it is not in the 3.18 branch.

File: calico-enterprise_versioned_docs/version-3.18/variables.js [hunk 7]
```diff
@@ -1,24 +1,24 @@
...
   windowsScriptsURL: 'https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess',
   prodnameWindows: 'Calico Enterprise for Windows',
   downloadsurl: 'https://downloads.tigera.io',
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
   rootDirWindows: 'C:\\TigeraCalico',
-  registry: 'gcr.io/unique-caldron-775/cnx/', // Change to 'quay.io/' for new release
-  chart_version_name: 'master',
+  registry: 'quay.io/',
+  chart_version_name: 'v3.18.0-0',
```
Comment (bmckercher123) [inline calico-enterprise_versioned_docs/version-3.18/variables.js:18]: @rene-dekker Is this correct for helm?

File: calico-enterprise_versioned_docs/version-3.18/releases.json (hunk 5 above)
Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.18/releases.json]: Yes. @bmckercher123 you'll need to move "csi-node-driver-registrar" back in line with the others. Should sort out your build issues.(At least it did for me locally.)

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18/variables.js (hunk 7 above)
Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.18/variables.js:18]: Yes

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by bmckercher123** [comment]
Note: Rene/Matt/Saba approved changing calico version to 3.26 on day of release.

File: calico-enterprise_versioned_docs/version-3.18/releases.json (hunk 3 above)
Comment (danudey) [reply calico-enterprise_versioned_docs/version-3.18/releases.json]: This was resolved per a discussion on slack.

**Review by danudey — APPROVED** [review]

**Comment by ctauchen** [comment]
Closing, was cloned in #911

**Pull request closed without merging**